Project.toml

```diff
@@ -4,7 +4,7 @@ authors = ["preCICE <https://precice.org/> and contributors"]
 version = "0.1.0"
 
 [compat]
-julia = "1"
+julia = "1.6"
```

Comment:
Is this sufficient to check for all the versions that we intend to check?

Reply:
This checks if the version is newer than `1.6`. I'm not sure how to check exactly the versions that are tested, [here](https://pkgdocs.julialang.org/v1/compatibility/#Version-specifier-format-1) is the version specifier guide. I think this would be enough, as it only would show a small message during installation. If we want to force the users, then we could add the supported versions to the manual check in [`build.jl`](https://github.com/precice/julia-bindings/blob/17aaa53de1409c8ddbe3adc1a237913979fd685a/deps/build.jl#L1-L3)

Reply:
Checking if the version is newer than `v1.6.0` is indeed sufficient here. Lets not block potential users who want to use some `beta` versions or something